Terminal.Gui/Core/View.cs

```diff
@@ -2303,28 +2391,20 @@ void CollectAll (View from, ref HashSet<View> nNodes, ref HashSet<(View, View)>
 		/// </para>
 		/// </remarks>
 		public virtual ustring Text {
-			get => TextFormatter.Text;
+			get => text;
 			set {
-				TextFormatter.Text = value;
-				var prevSize = frame.Size;
-				var canResize = ResizeView (autoSize);
-				if (canResize && TextFormatter.Size != Bounds.Size) {
-					Bounds = new Rect (new Point (Bounds.X, Bounds.Y), TextFormatter.Size);
-				} else if (!canResize && TextFormatter.Size != Bounds.Size) {
-					TextFormatter.Size = Bounds.Size;
-				}
-				SetMinWidthHeight ();
-				SetNeedsLayout ();
-				SetNeedsDisplay (new Rect (new Point (0, 0),
-					new Size (Math.Max (frame.Width, prevSize.Width), Math.Max (frame.Height, prevSize.Height))));
+				text = value;
+				SetHotKey ();
+				UpdateTextFormatterText ();
+				ProcessResizeView ();
 			}
 		}
 
 		/// <summary>
 		/// Used by <see cref="Text"/> to resize the view's <see cref="Bounds"/> with the <see cref="TextFormatter.Size"/>.
```

Comment:
The English of this documentation is not well-formed, and is thus hard to understand. I think it should say this:

```
Gets or sets a flag that determines whether the View will be automatically resized to fit the <see cref="Text"/>.  The default is `false`.  Set to `true` to turn on AutoSize. For <see cref="AutoSize"/> to work <see cref="Width"/> and <see cref="Height"/> must be null or a valid <see cref="LayoutStyle.Absolute"/> values. In addition, AutoSize only works with <see cref="LayoutStyle.Computed"/> layout,
```

Right?

Reply:
Thanks for pointing this. That not true anymore. It will look like this:

```cs
		/// <summary>
		/// Gets or sets a flag that determines whether the View will be automatically resized to fit the <see cref="Text"/>.
		/// The default is `false`. Set to `true` to turn on AutoSize. If <see cref="AutoSize"/> is `true` the <see cref="Width"/>
		/// and <see cref="Height"/> will always be used if the text size is lower. If the text size is higher the bounds will
		/// be resized to fit it.
		/// In addition, if <see cref="ForceValidatePosDim"/> is `true` the new values of <see cref="Width"/> and
		/// <see cref="Height"/> must be of the same types of the existing one to avoid breaking the <see cref="Dim"/> settings.
		/// </summary>
```